colors.scss

Leeyuryeon opened the pull request:
I found a typo in the document. This version of the documentation was more readable. Thank you

Leeyuryeon committed:
Update colors.scss

Leeyuryeon committed:
Update colors.scss

caiovaccaro commented:
Hi!

You can do style changes in your versions, we can't change the official version, sorry!

Leeyuryeon committed:
Update CONTRIBUTING.md

I found a typo in the document. This version of the documentation was more readable. Thank you.

Leeyuryeon committed:
Create README in Korean

The documents I translate into Korean will help many people. Thank you.

Leeyuryeon committed:
Update _data.json

improvement.

Leeyuryeon committed:
Update README.md

This method seems to be more useful. Thank you!
